pystac_io/__init__.py

```diff
@@ -0,0 +1,61 @@
+from collections import namedtuple
+from typing import Dict, List
+
+from pystac import STAC_IO
+
+"""Container for PSTAC_IO read write methods
+
+read: pystac.STAC_IO.read_text_method or None
+write: pystac.STAC_IO.write_text_method or None
+
+"""
+IoReadWrite = namedtuple("IoReadWrite", ["read", "write"])
+
+
+""" Private dict of registered IO modules """
+PYSTAC_IO: Dict[str, IoReadWrite] = {}
```

Comment:
This is a lot like what [`python-typeclasses`](https://github.com/thejohnfreeman/python-typeclasses/blob/master/typeclasses/decorator.py#L73-L82) does to track typeclass instances.